src/lib/css-tag.ts

```diff
@@ -12,7 +12,8 @@ found at http://polymer.github.io/PATENTS.txt
 /**
  * Whether the current browser supports `adoptedStyleSheets`.
  */
-export const supportsAdoptingStyleSheets =
+export const supportsAdoptingStyleSheets = (window.ShadowRoot) &&
+    (window.ShadyCSS === undefined) &&
```

Comment:
This can be defined but not in use... as in our test environment.

```suggestion
    (window.ShadyCSS === undefined || window.ShadyCSS.nativeShadow) &&
```